### PR TITLE
Fix: fixed code Checkbox / default empty value Input

### PR DESCRIPTION
**Pull request by FranciscoSanvicente**
# Fix: fixed code Checkbox

In the integration, an improvement in the Checkbox component was proposed, which were the following:

- Change in the names of the classes
- Remove inline styles
- Use custom properties

In the Input component the default value was changed from ' ' to ' ' because for security reasons, browsers do not allow to set a value different from ' ' (string without spaces) in file type inputs.

### CODE REVIEW

File: library/modules/form/controls/ts/checkbox/index.tsx [hunk 1]
```diff
@@ -6,27 +6,43 @@ export /*bundle*/ const Checkbox: React.FC<IProps & RefAttributes<HTMLInputEleme
...
 	const [value, setValue] = useState<boolean>(!!checked);
 	const handleChange = (event: React.ChangeEvent<HTMLInputElement>): void => {
 		event.stopPropagation();
-
-		setValue(!!checked);
+		setValue(!checked);
 		onChange && onChange(event);
 	};
-	let cls: string = `pui-checkbox ${className ? className : ''}`;
+	let cls: string = `pragmate-checkbox ${className ? className : ''}`;
```
Comment (jircdev) [inline library/modules/form/controls/ts/checkbox/index.tsx:16]: The class must be named "pui-". Please review it in any element.

File: library/modules/form/controls/ts/input/control.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ export function ControlSelector(): JSX.Element {
 	const attrs = {
 		id: id ?? name,
 		placeholder: placeholder ?? ' ',
```
Comment (jircdev) [inline library/modules/form/controls/ts/input/control.tsx:35]: Please remove the blank space.

File: library/modules/form/controls/scss/checkbox.scss [hunk 3]
```diff
@@ -1,5 +1,5 @@
 .pragmate-checkbox {
```
Comment (jircdev) [inline library/modules/form/controls/scss/checkbox.scss:1]: the css class must be named "pui" following the standard defined @FranciscoSanvicente

**Review by jircdev — CHANGES_REQUESTED** [review]

**Comment by jircdev** [comment]
@FranciscoSanvicente Did you manage "sizing" property for checkboxes ? Could be a good idea to manage different sizes like "xs", "sm" "lg". What do you think? like the inputs and other elements.

**Comment by FranciscoSanvicente** [comment]
I think it is a good idea for the component but I would leave it for a future version.

**Review by jircdev — APPROVED** [review]

**Pull request merged**